### DISCUSSION TITLE
Support p7zip-rar as faster alternative of unrar on some devices

### DISCUSSION THREAD

**Discussion by Prinz23** [nzbgetcom/nzbget#50] [Ideas]
On ARM devices unrar is extremely slow, specially if the rar is encrypted and/or compressed. 

p7zip-full and the p7zip-rar  as an alternative is magnitudes faster as seen in this thread:

[[Workaround] Slow unrar on Rpi4](https://forum.nzbget.net/viewtopic.php?t=3788)

I can confirm running nzbget with this script on a Banana Pi it's multiple times faster. 

It would be nice if nzbget could support this without the script workaround which has downsides like unpack time not been shown, errors not being handled in optimal way. As stated in the thread that workaround script only does the bare minimum to make nzbget work/happy.

**Comment by luckedea** [comment]
I'm noticing same while experimenting with docker setup as well, thank you for bringing this up. And it looks like something that can be easily and quickly integrated.
